cd/auto-generate/auto-generate-controllers.sh

```diff
@@ -158,6 +160,16 @@ for CONTROLLER_NAME in $CONTROLLER_NAMES; do
       fi
       echo "ok"
     fi
+
+    SERVICE_AVAILABLE_API_VERSION=$(yq e '.api_versions[] | select(.status == "available") | .api_version' metadata.yaml)
```

Comment:
nit: add an empty check for "SERVICE_AVAILABLE_API_VERSION" value and skip if this is empty